examples/s2s-api/sepa_payment.php

```diff
@@ -0,0 +1,125 @@
+<?php
+/**
+ * This example shows you how to process a Direct Debit (SEPA) transaction.
+ * You can use the payment detail token returned in this example to run the example in payment_with_saved_token.php
+ */
+
+include __DIR__ . '/../../vendor/autoload.php';
+
+try {
+    $creditCardPayment = new \VendoSdk\S2S\Request\Payment();
```

Comment:
The variable should be renamed to something else

Reply:
changed to $sepaPayment